governance/polls/Add wstETH-C as new Vault type - April 25, 2022.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Add wstETH-C as a new Vault Type - April 25, 2022
+summary: This poll proposes system parameters which could be used to initialize wstETH-C as a new vault type.
+discussion_link: https://forum.makerdao.com/t/introduction-risk-parameters-for-wsteth-c-vault/14669
+vote_type: Plurality Voting
+categories:
+   - Collateral
+   - Risk Variable
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-04-25T16:00:00
+end_date: 2022-04-28T16:00:00
+---
+# Poll: Add wstETH-C as a new Vault Type - April 25, 2022
+
+The Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the MakerDAO mandated actors. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, April 25 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll allows the MakerDAO governance community to signal their support or opposition to adding wstETH-C (Lido Staked ETH) as a new vault type in the Maker Protocol with the parameters below:
+
+### Initial Collateral Parameters
+
+* Underlying Collateral: wstETH
+* [Stability Fee](https://community-development.makerdao.com/en/learn/governance/param-stability-fee): 0.75%
+* [Liquidation Ratio](https://community-development.makerdao.com/en/learn/governance/param-liquidation-ratio): 185%
+* [Debt Ceiling (`line`)](https://community-development.makerdao.com/en/learn/governance/param-debt-ceiling): 150 million DAI
```

Comment:
For DC-IAM vaults, which this is, this should be "Maximum Debt Ceiling (`line`)"